doc/stdlib/base.rst

```diff
@@ -156,6 +156,10 @@ Getting Around
 
    This is only needed if your module depends on a file that is not used via ``include``\ . It has no effect outside of compilation.
 
+.. function:: apropos(string)
+
+   .. Docstring generated from Julia source
+
```

Comment:
@mbauman Can you check if the [yyc/genstdlib-module](https://github.com/JuliaLang/julia/tree/yyc/genstdlib-module) branch fixes it?